specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/phpWorkloads.json

```diff
@@ -0,0 +1,1362 @@
+          },
+          {
+            "name": "delete-infra",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Whether to delete infra along with workload resource.",
+            "minLength": 1
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Resource deleted."
+          },
+          "202": {
+            "description": "Resource deletion in progress."
+          },
+          "204": {
+            "description": "Resource not found."
+          },
+          "default": {
+            "description": "Resource provider error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Workloads": {
+            "$ref": "./examples/phpworkloads/PhpWorkloads_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/phpWorkloads/{phpWorkloadName}/wordpressInstances": {
+      "get": {
+        "tags": [
+          "WordpressInstances"
+        ],
+        "description": "Lists WordPress instance resources under a phpWorkload resource.",
+        "operationId": "WordpressInstances_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PhpWorkloadNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/wordpressInstanceResourceList"
+            }
+          },
+          "default": {
+            "description": "Resource provider error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Workloads": {
+            "$ref": "./examples/phpworkloads/WordpressInstances_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/phpWorkloads/{phpWorkloadName}/wordpressInstances/default": {
```

Comment:
Why "default"? is there any other wordpressInstance?

Reply:
No, we want to restrict number of WordPress instances to 1. Hence using "default" as the only instance.